Sources/Spezi/Spezi/View+Spezi.swift

```diff
@@ -26,13 +26,74 @@ struct SpeziViewModifier: ViewModifier {
 }
 
 
+/// Options to simulate behavior for a ``LifecycleHandler`` in cases where there is no app delegate like in Preview setups.
+public enum LifecycleSimulationOptions {
+    /// Simulation is disabled.
+    case disabled
+    /// The ``LifecycleHandler/willFinishLaunchingWithOptions(_:launchOptions:)-8jatp`` method will be called for all
+    /// configured ``Module``s that conform to ``LifecycleHandler``.
+    case launchWithOptions(_ launchOptions: [UIApplication.LaunchOptionsKey: Any])
```

Comment:
Could we add a default value of `[:]` here or add a static property that is called `launchWithOptions` that offers the default value so you could even write `.launchWithOptions`?